### ISSUE TITLE
Host scans don't work on Android 13 due to SELinux blocking nlmsg_getneigh

### ISSUE THREAD

**Issue by aaronjwood** [aaronjwood/PortAuthority#151]
```
type=1400 audit(0.0:21): avc: denied { nlmsg_getneigh } for scontext=u:r:untrusted_app:s0:c157,c256,c512,c768 tcontext=u:r:untrusted_app:s0:c157,c256,c512,c768 tclass=netlink_route_socket permissive=0 bug=b/155595000 app=com.aaronjwood.portauthority.donate
```

**Comment by Silent-Hunter** [comment]
I assume this is why the app crashes when I do Discover Hosts? Is there any way to fix this problem?

**Comment by aaronjwood** [comment]
Yeah, that's probably what you're hitting. The latest version I've rolled out (v2.4.5) works around this for now. It's not a long term solution as I am just targeting an older API version rather than the very latest.

**Comment by chimpzilla** [comment]
Has anyone found a workaround for gathering mac addresses? Google Play won't let you publish now unless you target API33. It's so daft, because it means you can't know which device is which on a local network. I think Apple maybe has a similar policy and that Google have simply copied it, without properly thinking through the consequences.
